mozilla-mobile/mozilla-vpn-client
# IGNORE - Basti does wierd taskcluster stuff again

- Addons for 2.9.0 (#3755)
- Add level-3 latest-builds to a version-keyed index (#3761)
- ci: Add missing 'index' prefix to Taskcluster branch routes (#3787)
- Add missing entitlements (#3785)
- Fixing xcode cloud (#3784)
- Glean events for subscription management and account deletion (#3764)
- Inspector command to force subscription management re-authentication (#3782)
- No Bug - Small nits to fix taskcluster issues
- Support languages for add-ons - fixes #VPN-2421 and #VPN-2417
- Use the Glean-Android SDK on instead of glean.js on android (#3375)
- Add periodic check (3 hours / 1 minute) for controller state (#3717)
- Fix parsing of subscription payment data (#3793)
- Remove RPM github action (#3809)
- Disable wheel events during the tutorials - VPN-2433
- Change the order of priority for the XLIFF and TS files
- Support back-button on android in the guide view - fixes VPN-2435
- Glean: Add Host's internet connection type to glean events (#3651)
- Add periodic timer to android (#3724)
- Fix Missing Text in segmented button (#3815)
- Make windows ci more resilient (#3768)
- Support unique survey alert for 2.9 UR #3402 / VPN-2139 - WIP  (#3756)
- Import languages before generating addons
- Expose the alert message using the correct VPNSurveyModel property
- Add fallback path when pkg-config is unable to find systemd (#3826)
- Enable subscription management feature by default (#3800)
- Fix #3811 - Leave tutorial modal issues when opened from top bar (#3823)
- Fix IAA error messages on Android #3279 (#3804)
- feature tour and guide view layout fixes (#3825)
- Support back-button on android in the guide view - fixes VPN-2435 (#3824)
- Add QEvent::TouchEnd to ItemPicker::eventFilterInternal (#3834)
- Show tooltips only on hover during a tutorial (#3820)
- Always reset profile flow before starting it (#3836)
- Deactivate the VPN when starting the first tutorial (#3838)
- Go back to the previous onboarding view from the get-help menu - VPN-2258 (#3832)
- Parse subscription data depending on subscription type (#3833)
- Add more tests for the auth-in-app (#3781)
- Device limit functional test - VPN-2414 (#3767)
- Properly disable qmljsdebugger in release mode (#3803)
- Use whole sliding-window when calulating Ping-Loss (#3412)
- Sub management re-authentication flow enhancements (#3772)
- Optimize the item picker same-event callback (#3818)
- Make the flickable items not scrollable during a tutorial (#3845)
- Reset onClosed() when opening "Leave tutorial?" popup (#3846)
- Better locale filtering for surveys - VPN-2464 (#3860)
- Logout on account deletion (#3843)
- Delete tasks when the account is deleted - fixed VPN-2463 (#3861)
- chore: Manage subscription links (#3813)
- Fixing the unit-test compilation
- Handling external operations in one single place (#3865)
- Use interval to display the correct subscription plan period (#3849)
- Remove the screen-capture github action + test - we are not using it (#3819)
- Fix subscription payment parsing for IAP (#3872)
- Fix subscription management for Apple subscriptions (#3874)
- Add tests for the auth in subscription and account deletion (#3876)
- chore: Support more intervals for testing (#3877)
- Remove extra files in the final addon folder (#3890)
- Add new Telemetry for sku investigation (#3759)
- Fixes app background color and divider color on Android (#3827)
- Remove extra files in the final addon folder (#3890)
- Restore copy/paste functionality for inputs on iOS (#3875)
- Let's schedule the support task immediately to avoid deadlocks - fixes VPN-2481 (#3894)
- Inform the user when the auth fails in account-deletion and subscription view - VPN-2482 (#3896)
- fix: Force profile re-auth inspector command (#3870)
- Support close-event on android - VPN-2467 (#3889)
- Retranslate the server countries when the language changes - VPN-2484
- Make the user profile button not clikable too much - VPN-2485
- fix: Profile arrow button (#3911)
- Handle debit and prepaid cards in subscription management (#3908)
- fix: Clear subscription data before populating (#3914)
- Do not show plan data for Google subscriptions (#3929)
- fix: Vertical subscription alignment (#3932)
- No multiple tips&tricks view when clicking multiple times the popup button - VPN-2506 (#3940)
- Fix the windows installer path for 2.9 (#3943)
- Partially revert  "Fix the windows installer path for 2.9 (#3943)"
- tips and tricks telemetry (#3839)
- Fix merge issues for the tips&tricks telemetry
- Open FxA directly to show the account management URL - VPN-2549 (#3967)
- Consume signed stuff from the 'signed-internals' folder (#3969)
- fix: Text field focus and navigation (#3970)
- Enable MVPN_WEBEXTENSION when building for desktop (#3971)
- fix: Crash on swipe kill (#4005)
- Switch run condition (#4017)
- Quit the app asap without calling the DTOR (#4016)
- HackFix for #4002 / 2.9 only (#4018)
- Hotfix: Glean for 2.9
- Make subscription data sensitive in logs (#4027)
- Hotfix: Bump windows toolchain version
- De-Prio 64bit builds

## Description

    Describe your changes

## Reference

    i.e Jira or Github issue URL

## Checklist
    
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have added thorough tests where needed
